admissioncontroller/k8s/deployment.yaml

```diff
@@ -59,7 +60,12 @@ spec:
         - name: BC_SOURCE
           value: admissionController
         - name: CKV_GITHUB_CONFIG_FETCH_DATA
-          value: "False"
+          value: "False"     
+        - name: DEBUG
+          valueFrom:
+            configMapKeyRef:
+              name: whorfconfig # The ConfigMap this value comes from.
+              key: debug        # Are we in debug mode?                     
```

Comment:
do we want to keep it in debug?

Reply:
Debug status is dictated by the config map which is by default 'no'